modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -26,7 +27,10 @@ def apply_list_of_funcs(funcs, df):
         if isinstance(func, bytes):
             func = pkl.loads(func)
         df = func(df, **kwargs)
-    return df, get_ip()
+    if EnablePartitionIPs.get():
```

Comment:
What is the use case for getting the IP of the partition each time? I prefer not adding this to the existing `deploy_func` rather add a new function specific to this. Having multiple return types from this function can only invite more bugs.

Reply:
The use case is that an user can call `unwrap_partitions(df, axis=None, bind_ip=True) at any time point during his workflow and IPs of the partitions have to be valid.

Reply:
I am not sure on what bugs you are saying about but we can add an additional function to return multiple values.

Reply:
We should just always return ip as a part of the metadata in that case instead of optionally. The user can optionally see it, but it will always be there.

If a user changes `EnablePartitionIPs` in the middle of a workflow, I believe it can cause some problems if we conditionally return different numbers of objects. That is the bug I was talking about.

Reply:
Yes, that is true on the bug you were saying about. So we could add a note on `EnablePartitionIPs` changing during a workflow. However, it looks not user-friendly. Let's always return ip as a part of the metadata then.